backend/src/main/java/com/f12/moitz/domain/subway/SubwayRouteCalculator.java

```diff
@@ -74,11 +104,14 @@ public StationSequence findShortestTimePath(final SubwayStation start, final Sub
 
                         if (transferEdge.isEmpty()) {
                             log.warn(
-                                    "환승 Edge가 존재하지 않아 경로를 건너뜁니다. 현재역: {}, 다음역: {}, 환승호선: {} -> {}",
+                                    "환승 Edge가 존재하지 않아 경로를 건너뜁니다. 현재역: {}, 다음역: {}, 환승호선: {} -> {} "
```

Comment:
엇 이거는 + 연산 안 하더라도 해결 가능한 것 같은데 확인해주실 수 있을까욥

```suggestion
                                    "환승 Edge가 존재하지 않아 경로를 건너뜁니다. 현재역: {}, 다음역: {}, 환승호선: {} -> {} (탐색경로 - 출발역: {}, 도착역: {})"
```

Reply:
아 이거 + 연산이 필요한 건 아닌데 로그 메시지가 너무 길어져서 줄바꿈 이슈로 메시지를 나눠봤어요..ㅋㅋ 근데 이런건 그냥 줄바꿈 글자제한 안 지키고 놔둬도 되려나요~?

Reply:
아하 이해했어요! 왠지 이으려는 의도는 아닌 것 같긴 했어요 ㅋㅋㅋㅎ

그런 이유라면 텍스트 블록 `"""`을 사용하여 줄바꾸는 방식은 어떤 것 같아요?

Reply:
헐 뭐지 아까 텍스트 블록 했을 때는 플레이스 홀더가 적용이 안됐었는데 다시 하니까 되네여?!?!
아까는 제가 뭘 잘못 했었나바여... 텍스트블록 방식으로 바꿀게여!ㅎㅎ